tests/foreman/cli/test_contentview.py

```diff
@@ -1074,6 +1074,92 @@ def test_negative_add_same_puppet_repo_twice(self):
                     u'name': puppet_module['name'],
                 })
 
+    @tier2
+    @run_only_on('sat')
+    def test_negative_add_unpublished_cv_to_composite(self):
+        """Attempt to associate unpublished non-composite content view with
+        composite content view.
+
+        @id: acee782f-2792-4c4e-b0c9-87d6b89992ef
+
+        @steps:
+        1. Create an empty non-composite content view. Do not publish it.
+        2. Create a new composite content view
+
+        @assert: Non-composite content view cannot be added to
+        composite content view.
+
+        @CaseLevel: Integration
+
+        @BZ: 1367123
+        """
+        # Create component CV
```

Comment:
You mean non-composite? Composite CV can contain components -- other non-composite CV.